src/lib/components/Cytoscape.react.js

```diff
@@ -296,6 +390,10 @@ class Cytoscape extends Component {
                 }),
             });
         });
+        cy.contextMenus({
```

Comment:
@cleaaum Have you tested updating the `contextMenu` prop with a callback, and verified that the menu items change as expected?

I *think* it will work given the way you've implemented it here, but worth testing.

Reply:
I was just about to ask the same thing - `handleCy` seems to be called implicitly by `react-cytoscapejs` so my suspicion is no prop changes will cause it to re-execute... if that's the case it'll be a bigger project to fix it, but also re: the `if (length > 0)` you just added, once it's generally working it'll be important to check whether clearing it entirely works or leaves the previous menu in place.